### DISCUSSION TITLE
onSaveTaskOutputItem not reacting

### DISCUSSION THREAD

**Discussion by Melina-Jingting** [UniversalDataTool/universal-data-tool#418] [General]
Hello! I'm trying out integration with javascript library - https://docs.universaldatatool.com/integrate-with-any-web-page/integrate-with-the-javascript-library
However when I click save, onSaveTaskOutputItem does not seem to be triggered to output anything. 
I followed the docs as is, only changing to text_entity_recognition type and the input data

**Comment by seveibar** [comment]
hmm this is definitely concerning, sorry for seeing this so late
